### PR TITLE
Added new url shortener (Simple-URL-Shortener)

### PR DESCRIPTION
**Pull request by azlux**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
- [X] Your additions are ordered alphabetically.
- [X] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [X] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [X] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [X] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [X] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/Kickball/awesome-selfhosted/issues) or [PRs](https://github.com/Kickball/awesome-selfhosted/pulls), including closed ones.
- [X] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [X] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.

Az

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1362,6 +1362,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [schort](https://github.com/sqozz/schort) - No login, no javascript, just short links. ([Demo](https://s.wx0.de)) `CC0-1.0` `Python`
 - [shorturl](https://github.com/prologic/shorturl) - Simple URL shortener with very tiny URLs. ([Demo](https://url.mills.io)) `MIT` `Go`
 - [shuri](https://github.com/pips-/shuri) - SHort URI - Lighweight URL shortener. `MIT` `PHP`
+- [Simple-URL-Shortener](https://github.com/azlux/Simple-URL-Shortener) -  KISS URL shortener, public or private (with account). Minimalist and lighweight. No dependancies ([Demo](https://u.azlux.fr)) `MIT` `PHP`
```
Comment (nodiscc) [inline README.md]: Depend**e**ncies is teh correct spelling.

Missing final dot.

File: README.md [hunk 2]
```diff
@@ -1362,6 +1362,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 - [schort](https://github.com/sqozz/schort) - No login, no javascript, just short links. ([Demo](https://s.wx0.de)) `CC0-1.0` `Python`
 - [shorturl](https://github.com/prologic/shorturl) - Simple URL shortener with very tiny URLs. ([Demo](https://url.mills.io)) `MIT` `Go`
 - [shuri](https://github.com/pips-/shuri) - SHort URI - Lighweight URL shortener. `MIT` `PHP`
+- [Simple-URL-Shortener](https://github.com/azlux/Simple-URL-Shortener) -  KISS URL shortener, public or private (with account). Minimalist and lighweight. No dependencies ([Demo](https://u.azlux.fr)) `MIT` `PHP`
```
Comment (nodiscc) [inline README.md]: Please remove the double space

Comment (nodiscc) [reply README.md]: Please add the missing final dot

Comment (azlux) [reply README.md]: Srry for that

**Pull request merged**